hub-saml/src/main/java/uk/gov/ida/saml/hub/validators/response/common/UnDecryptedValidatedResponseCounter.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.ida.saml.hub.validators.response.common;
+import uk.gov.ida.saml.security.validators.ValidatedResponse;
+
+public interface UnDecryptedValidatedResponseCounter {
```

Comment:
Not that keen on the name of this interface given it represents a container that holds a response, and places a metric as a side effect. The suffix Counter makes it feel like its some kind of Counter.

Reply:
Removed this interface entirely